client/src/dev-components/container-card/container-card.js

```diff
@@ -5,32 +5,31 @@ import InlineSVG from 'svg-inline-react'
 // Helpers
 import classNames from 'classnames/bind'
 import apiCall from '../../api-call'
+import { shareSlack } from '../../globals'
 // Styles
 import styles from './container-card.scss'
+import '../../scss/atoms/_toast.scss'
 // Icons
 import CloseIcon from '../../assets/svg/baseline-close-24px.svg'
 import StartIcon from '../../assets/svg/baseline-play_arrow-24px.svg'
 import StopIcon from '../../assets/svg/baseline-stop-24px.svg'
+import SlackIcon from '../../assets/svg/baseline-slack-24px.svg'
 
 const css = classNames.bind(styles)
 
 class ContainerCard extends React.Component {
   constructor(props) {
     super(props)
     this.instance = this.props.instance
+    this.instance.url = `//${this.instance.subdomain}.${window.location.host}`
     this.instance.name = this.instance.subdomain
 
-    this.instanceUrl = `//${this.instance.subdomain}.${window.location.host}`
-    const { blob } = this.instance
-    if (blob.path) {
-      this.instanceUrl += blob.path
-    }
```

Comment:
This should not be removed as it breaks some external instances!
Merge issue?

Reply:
reverted

Reply:
reverted